service-front/composer.json

```diff
@@ -10,11 +10,12 @@
     ],
     "homepage": "https://github.com/ministryofjustice/opg-lpa",
     "require": {
-        "php": ">=8.1 <8.2.0",
-        "guzzlehttp/guzzle": "^6.3",
+        "php": "^8.1",
+        "php": ">=8.1 <8.2.2",
```

Comment:
ditto, we only want PHP < 8.2.0 for service-front for now

Reply:
OK sorted